### PR TITLE
Hide "Show My Work" when we can't run a report.

### PR DESCRIPTION
**Pull request by knowuh**
On the completion page, we gray out/disable the "Show My Work" button unless we can run a report.

This should have been a small change, but I got derailed by a failing test, and then remembered that we were supposed to disable a button instead of removing it.

[#177461358] -- Hide "Show My Work" when we can't run a report

PT Story: https://www.pivotaltracker.com/story/show/177461358

### CODE REVIEW

**Review by scytacki — APPROVED** [review]
looks good to me

**Comment by knowuh** [comment]
@scytacki sorry -- I added one more commit.  I forgot that instead of removing the button we were supposed to just disable it.  Attached are the screen shots from the 4 states we could be in:

- Online
- Online but in preview mode
- Offline with tokens
- Offline without Tokens
Screen Shot 2021-03-24 at 2 42 55 PM
Screen Shot 2021-03-24 at 2 42 36 PM
Screen Shot 2021-03-24 at 2 42 20 PM
Screen Shot 2021-03-24 at 2 41 39 PM

File: src/components/activity-completion/completion-page-content.tsx [hunk 1]
```diff
@@ -165,10 +167,25 @@ export const CompletionPageContent: React.FC<IProps> = (props) => {
...
+                      className="button show-my-work disabled"
+                      disabled={true}>
+                        <IconCompletion width={24} height={24} />
+                        Show My Work
+                    </button>
+              }
+            {(!sequence || isLastActivityInSequence) &&
               <div className="exit-button">
-                <span>or</span>
+                { showStudentReportButton && <span>or</span> }
```
Comment (scytacki) [inline src/components/activity-completion/completion-page-content.tsx:188]: Seems like the `or` should be shown all of the time now that the button is just disabled.

**Review by scytacki — CHANGES_REQUESTED** [review]
one issue with the "or" text

**Comment by knowuh** [comment]
I was wondering about that "or" button.  Any way, its back in.

Screen Shot 2021-03-24 at 3 35 21 PM
Screen Shot 2021-03-24 at 3 34 28 PM

**Review by scytacki — APPROVED** [review]
Looks good.

**Pull request merged**